### DISCUSSION TITLE
Preventing Ctrl+Mouse zooming

### DISCUSSION THREAD

**Discussion by guyas** [SafeExamBrowser/seb-win-refactoring#873] [Q&A]
Hi,
I am using version 3.6.0 for windows and have encountered something which I believe should not happen:
Although zoom in not enabled (under UI > Enable zoom, both boxes are unchecked), the users still manage to zoom in/out by using Ctrl+ mouse wheel. Is there any way of blocking it?

**Comment by dbuechel** [comment]
At the moment unfortunately not. We are tracking this issue internally, but it is in the end an upstream issue of the browser engine (see https://github.com/cefsharp/CefSharp/discussions/4732 and https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=19768).

**Reply by guyas** [reply]
OK, I hope this will be resolved in some future version. Thanks anyway!

**Reply by dbuechel** [reply]
Yes, we'll definitely try to find a solution for an upcoming version, potentially even for SEB 3.8.0 if time permits.
